modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -55,33 +55,32 @@ $ coreos-installer install --copy-network \
 [id="installation-user-infra-machines-advanced_disk_{context}"]
 == Disk partitioning
-In most cases, the {product-title} installer should be allowed to configure
-your disk partitions. However, there are two cases where you might want to
-intervene to override the default partitioning when installing an
+In most cases, data partitions are originally created by installing {op-system}, rather than by installing another operating system. In such cases, the {product-title} installer should be allowed to configure your disk partitions.
+
+However, there are two cases where you might want to intervene to override the default partitioning when installing an
 {product-title} node:
 
 * Create separate partitions: For greenfield installations on an empty
-disk, you might want to add separate storage to a partition. This is only
-officially supported for making `/var` or a subdirectory of `/var` a separate
-partition, but not both. If you create more than one partition, Kubernetes
-will not be able to monitor them both.
+disk, you might want to add separate storage to a partition. This is
+officially supported for making `/var` or a subdirectory of `/var`, such as `/var/lib/etcd`, a separate partition, but not both.
++
+Kubernetes supports only two filesystem partitions. If you add more than one partition to the original configuration, Kubernetes cannot monitor all of them.
 
-* Retain existing partitions: For a brownfield installation, where you are reinstalling on an existing {product-title} node and want to retain data partitions from your previous operating system, there are both boot arguments and options to `coreos-installer` that allow you to retain existing data partitions.
+* Retain existing partitions: For a brownfield installation where you are reinstalling {product-title} on an existing node and want to retain data partitions installed from your previous operating system, there are both boot arguments and options to `coreos-installer` that allow you to retain existing data partitions.
 
 [id="installation-user-infra-machines-advanced_vardisk_{context}"]
 === Creating a separate `/var` partition
 In general, disk partitioning for {product-title} should be left to the
-installer. However, there are cases where you might want to create a
-separate partition in a part of the file system that you expect to grow.
+installer. However, there are cases where you might want to create separate partitions in a part of the filesystem that you expect to grow.
 
 {product-title} supports the addition of a single partition to attach
 storage to either the `/var` partition or a subdirectory of `/var`.
 For example:
 
 * `/var/lib/containers`: Holds container-related content that can grow
 as more images and containers are added to a system.
-* `/var`: Holds data that you might want to keep separate for purposes such as
-auditing.
+* `/var/lib/etcd`: Holds data that you might want to keep separate for purposes such as performance optimization of etcd storage.
```

Comment:
This is added based on a request in [BZ 1898275](https://bugzilla.redhat.com/show_bug.cgi?id=1898275):